sections/infrastructure.include

```diff
@@ -3908,7 +3908,7 @@ interface NoncedElement {
 
   <p class="note">As each {{Document}}'s [=response/CSP list=] is append-only, user agents can optimize away the <a>contains a header-delivered
   Content Security Policy</a> check by, for example, holding a flag on the {{Document}},
-  set during <a>Document initialization</a>.</p>
+  set during Document initialization.</p>
```

Comment:
Should the reference be to https://w3c.github.io/html/browsers.html#initializing-a-new-document-object - maybe adding an alternate term there would be better.

or `set when <a>initializing a new document object</a>` ?

Reply:
Done.